Source/WebCore/animation/ViewTimeline.cpp

```diff
@@ -203,4 +199,16 @@ ViewTimeline::Data ViewTimeline::computeViewTimelineData() const
     return { scrollContainerSize, subjectOffset, currentScrollOffset, coverRangeEnd };
 }
 
+const CSSNumericValue& ViewTimeline::startOffset() const
+{
+    auto viewTimelineData = computeViewTimelineData();
```

Comment:
I think just `data` is enough given the context.

Reply:
Fixed.